ui/components/app/assets/nfts/nft-details/nft-details.tsx

```diff
@@ -292,7 +293,7 @@ export function NftDetailsComponent({
         type: AssetType.NFT,
         details: {
           ...nft,
-          tokenId: Number(nft.tokenId),
```

Comment:
For long token IDs, this led to incorrectly encoded parameters for the `eth_estimateGas` query, ultimately causing the button to send the NFT to be disabled.

Reply:
Nice find!!

(future) - It would be nice if this type was corrected (instead of a `number` we keep it as a `string`. Or maybe make this a `string | number`?)